saleor/wsgi/__init__.py

```diff
@@ -34,8 +27,9 @@ def get_allowed_host_lazy():
 application(
     {
         "REQUEST_METHOD": "GET",
-        "SERVER_NAME": SimpleLazyObject(get_allowed_host_lazy),
+        "SERVER_NAME": "127.0.0.1",
```

Comment:
What if this is not an allowed hostname?

Reply:
If that's not allowed it is fine - it will say the host is not allowed but Django is still hit and got warmed up.
I can add a comment for it